### ISSUE TITLE
Typhoon & CV patching

### ISSUE THREAD

**Issue by TotalRefusal** [jakplugg/Typhoon#8]
Hi there!
I noticed a strange behavoir when patching CV into Typhoon. All 4 slider led´s are flashing up in the moment when I plug the jack in. Is that normal (because of the AUX input feature) or did I miss something??
Cheers

**Comment by TotalRefusal** [comment]
... this is the case with all CV inputs except V/Oct, Freeze and both VCA inputs

**Comment by TotalRefusal** [comment]
Yeah, it is that AUX CV feature connected to the switches of the CV input jacks. Took a look @schematics. This clicks are also clearly audible of course, if patching cables are "hot"

Is there a workaround for that???

**Comment by jakplugg** [comment]
Maybe the aux op amp or surrounding circuit is picking up the noise.
It was unaltered from the microcell (not sure if that was directly
copied from supercell) I will take a look at it.

**Comment by TotalRefusal** [comment]
It´s a mechanical thing when plugging in the patching cable, I think. Happens only, when patching from a CV source into Typhoon. So when you plug in, you´are touching AUX CV bus for a fraction of a second, I suppose....

**Comment by TotalRefusal** [comment]
Only solution I can see is patching from Typhoon to CV source and not vice versa. Or kick out that AUX CV bus feature... which makes no sense to me anyway :)

**Comment by frekkia** [comment]
I have a problem with typhoon with the Spectral Clouds (Kammerl) mode, typhoon stops after a short while I use it what can it be?
